### PR TITLE
Add increment operator

### LINKED ISSUE #15: Mock FieldValue.increment()
Firebase SDK [5.9.0](https://firebase.google.com/support/release-notes/js#version_590_-_mar_14_2019) added this neat new method that we should mock.

### PR DESCRIPTION
**Pull request by alexlouden**
Fixes #15

### CODE REVIEW

File: test/unit/firestore-document.js [hunk 1]
```diff
@@ -214,6 +229,23 @@ describe('MockFirestoreDocument', function () {
...
+      doc.set({
+        titles: 10
+      });
+      doc.set({
+        titles: Firestore.FieldValue.increment(10)
+      });
+
+      doc.get().then(function (snap) {
+        expect(snap.exists).to.equal(true);
+        expect(snap.data()).to.deep.equal({titles: 10});
```
Comment (dmurvihill) [inline test/unit/firestore-document.js]: Shouldn't this be `titles: 20`? Since we set it to 10 and then incremented it by 10?

File: src/utils.js [hunk 2]
```diff
@@ -155,14 +155,21 @@ exports.removeEmptyFirestoreProperties = function removeEmptyFirestoreProperties
...
-      }
-      if (FieldValue.arrayUnion().isEqual(value)) {
+      } else if (FieldValue.arrayUnion().isEqual(value)) {
         const replacement = Array.isArray(value.arg) ? value.arg : [value.arg];
         obj[s] = _.union(current[s], replacement);
+      } else if (FieldValue.increment().isEqual(value)) {
+        if (current[s] == null) {
+          // no existing data
+          obj[s] = value.arg;
+          console.log(obj[s], value);
```
Comment (dmurvihill) [inline src/utils.js]: Stray console log here

Comment (alexlouden) [reply src/utils.js]: Whoops, thanks - I've removed it

**Comment by alexlouden** [comment]
Done, thank you @dmurvihill!

**Comment by dmurvihill** [comment]
Thanks, great contribution!

**Pull request merged**